### DISCUSSION TITLE
Weekly Update 06/29/2022

### DISCUSSION THREAD

**Discussion by dinaolmelak** [Swarmies/sap-core#47] [General]
# Dinaol

###  How do you feel today?

- Feeling good.

### What did you do since the last meeting? 

- I worked on ticket #39. I created PR and #46 

### What are you planning to do moving forward? 

- Moving forward I plan to assist my fellow developers with their tickets so we can reach our milestone.

### Anything hampering your process? 

- Nothing is hampering my progress.

**Comment by kirubeltadesse** [comment]
# Kirubel

###  How do you feel today?

- Calm 🕊️ 

### What did you do since the last meeting? 

- I tried different ways of configuring the Circle CI. 
- PR viewed #46 

### What are you planning to do moving forward? 

- I feel like I have understood what the problem was. I am planning to roll out the fix and start working on the remaining tickets #41 and #28 

### Anything hampering your process? 

- Not enough time the last past few days. Since I am done with my paperwork, I will have some time this week hopefully.
